Formula/aspectj.rb

```diff
@@ -3,16 +3,17 @@ class Aspectj < Formula
   homepage "https://eclipse.org/aspectj/"
   url "https://www.eclipse.org/downloads/download.php?r=1&file=/tools/aspectj/aspectj-1.9.6.jar"
   sha256 "afec62c03fe154adeecf9cd599ce033fff258d1d373a82511e5df54f79ab03e2"
+  revision 1
 
   bottle :unneeded
 
-  depends_on java: "1.8"
+  depends_on "openjdk"
 
   def install
     mkdir_p "#{libexec}/#{name}"
-    system "java", "-jar", "aspectj-#{version}.jar", "-to", "#{libexec}/#{name}"
+    system "#{Formula["openjdk"].bin}/java", "-jar", "aspectj-#{version}.jar", "-to", "#{libexec}/#{name}"
     bin.install Dir["#{libexec}/#{name}/bin/*"]
-    bin.env_script_all_files(libexec/"#{name}/bin", Language::Java.java_home_env("1.8"))
+    bin.env_script_all_files libexec/"#{name}/bin", JAVA_HOME: "${JAVA_HOME:-#{Formula["openjdk"].opt_prefix}}"
```

Comment:
Would `Language::Java.overridable_java_home_env` make sense in this case?

Reply:
Good catch. I didn't know that it existed. 😅